test/test-file.js

```diff
@@ -16,15 +16,16 @@ describe('File', function () {
     assert.ok(file.blocks.length, 2);
   });
 
-  it('should *not* skip blank lines', function () {
+  it('should skip blank lines', function () {
     var filename = __dirname + '/fixtures/block_with_empty_line.html';
```

Comment:
@marcelaraujo Could you rebase on dev head and change `__dirname + '/fixtures/block_with_empty_line.html'` to 
`path.join(__dirname, '/fixtures/block_with_empty_line.html')`.
This should help us keep in sync with master.